logprep/processor/amides/detection.py

```diff
@@ -88,14 +88,14 @@ def detect(self, sample: str) -> int:
 
         Returns
         -------
-        predict: int.
-            0 or 1 if classified as benign or malicious.
+        predict: Tuple[boolean, float].
+            Detection result as boolean value and the corresponding confidence.
         """
         confidence_value = super().detect(sample)
 
         if confidence_value >= self._decision_threshold:
-            return 1
-        return 0
+            return True, round(confidence_value, 3)
+        return False, round(confidence_value, 3)
```

Comment:
Could you also please adjust the method return type in line 80?

Reply:
Of course :)